README.md

```diff
@@ -27,13 +27,26 @@ var css = fs.readFileSync("input.css", "utf8")
 
 var out = postcss().use(prefix({
   prefix: '.some-selector ', // <--- notice the traililng space!
-  exclude: ['.c']
+  exclude: ['.c'],
+
+  // Optional transform callback for case-by-case overrides 
+  transform: function (prefix, selector, prefixAndSelector) {
```

Comment:
Would be awesome if you could add an extra paragraph at the end under [`Options`](https://github.com/RadValentin/postcss-prefix-selector#options) about this.